### PR TITLE
ADCM-1937 fix bug with job_list

### PR DESCRIPTION
**Pull request by SofiaTal**
Need advice about correct behaviour of job_list.

### CODE REVIEW

File: python/cm/migrations/0068_alter_tasklog_action.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Generated by Django 3.2.4 on 2021-08-05 14:31
+# pylint: disable=line-too-long
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```
Comment (acmnu) [inline python/cm/migrations/0068_alter_tasklog_action.py:19]: @SofiaTal What about existing database with incorrect value of 'action' field, or this issue was not released?

**Review by acmnu — CHANGES_REQUESTED** [review]

File: python/cm/migrations/0068_alter_tasklog_action.py (hunk 1 above)
Comment (SofiaTal) [reply python/cm/migrations/0068_alter_tasklog_action.py:19]: I think that adding migration solve this problem, isn't it?

File: python/cm/migrations/0068_alter_tasklog_action.py [hunk 2]
```diff
@@ -15,6 +15,15 @@
 from django.db import migrations, models
 import django.db.models.deletion
 
+def fix_joblog(apps, schema_editor):
+    JobLog = apps.get_model('cm', 'JobLog')
+    for job in JobLog.objects.all():
```
Comment (nagarin) [inline python/cm/migrations/0068_alter_tasklog_action.py]: should be single query, something like `JobLog.objects.filter(task=None).delete()`

**Review by nagarin — CHANGES_REQUESTED** [review]

File: python/cm/migrations/0068_alter_tasklog_action.py (hunk 2 above)
Comment (SofiaTal) [reply python/cm/migrations/0068_alter_tasklog_action.py]: It is done in last commit before your review

**Review by nagarin — APPROVED** [review]

**Comment by nagarin** [comment]
https://stackoverflow.com/questions/38641952/django-filter-a-foreignkey-field-when-it-is-null

File: python/cm/migrations/0068_alter_tasklog_action.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def fix_joblog(apps, schema_editor):
+    JobLog = apps.get_model('cm', 'JobLog')
+    LogStorage = apps.get_model('cm', 'LogStorage')
+    for job in JobLog.objects.filter(task_id__isnull=True):
+        LogStorage.objects.filter(job_id=job.id).delete()
```
Comment (acmnu) [inline python/cm/migrations/0068_alter_tasklog_action.py:23]:
```suggestion
        LogStorage.objects.filter(job__task_id__isnull=True).delete()
```

**Review by acmnu — CHANGES_REQUESTED** [review]

**Review by nagarin — APPROVED** [review]

**Review by a-alferov — APPROVED** [review]

**Review by acmnu — APPROVED** [review]

**Pull request merged**